SDKAndroid/src/main/java/app/cybrid/sdkandroid/components/ListPricesView.kt

```diff
@@ -333,7 +336,7 @@ fun CryptoAssetItem(crypto: SymbolPriceBankModel,
                     index:Int, selectedIndex:Int,
                     context: Context? = null,
                     customStyles: ListPricesViewCustomStyles,
-                    onClick: (Int) -> Unit) {
+                    onClick: (AssetBankModel, AssetBankModel) -> Unit) {
```

Comment:
Could you add labels to the closure properties? It's not self explanatory on the purpose of each property.